unit-tests/live/d400/test-hdr.cpp

```diff
@@ -519,6 +519,11 @@ TEST_CASE("HDR Streaming - checking sequence id", "[hdr][live][using_pipeline]")
                         else
                         {
                             sequence_id = (sequence_id == 0) ? 1 : 0;
+                            if (sequence_id != depth_seq_id)
```

Comment:
You can use the `catch2` integrated feature called `TRACE`.
e.g. https://github.com/IntelRealSense/librealsense/blob/b874e42685aed1269bc57a2fe5bf14946deb6ede/unit-tests/algo/thermal-loop/test-get-scale.cpp#L38

It will only print if next `REQUIRE` will fail, so no logic needed.